docs/user/resources/04-50-10-aws-redis-cluster.md

```diff
@@ -0,0 +1,85 @@
+# AwsRedisCluster Custom Resource
+
+The `awsrediscluster.cloud-resources.kyma-project.io` is a namespace-scoped custom resource (CR).
+It describes the AWS ElastiCache Redis instance with cluster mode enabled.
+After the instance is provisioned, a Kubernetes Secret with endpoint and credential details is provided in the same namespace.
+By default, the created auth Secret has the same name as the AwsRedisCluster, unless specified otherwise.
+
+The AwsRedisCluster requires an IpRange. The size of IpRange is relative to the number of shards and replicas. Those IP addresses are allocated from the [IpRange](./04-10-iprange.md).
+If the IpRange is not specified in the AwsRedisCluster, the default IpRange is used.
+If a default IpRange does not exist, it is automatically created.
+
+When creating AwsRedisCluster, the `redisTier`, and `shardCount` fields are mandatory.
+
+Optionally, you can specify the `replicasPerShard`, `engineVersion`, `authEnabled`, `parameters`, and `preferredMaintenanceWindow` fields.
+
+As in-transit encryption is always enabled, communication with the Redis instance requires a trusted Certificate Authority (CA). You must install it on the container (e.g., using `apt-get install -y ca-certificates && update-ca-certificate`).
+
+## Redis Cluster Tiers
+
+| RedisTier | Capacity (GiB) | Network (up to Gbps) | Machine            |
+| --------- | -------------- | -------------------- | ------------------ |
+| C1        | 1.37           | 5                    | cache.t4g.small    |
+| C2        | 3.09           | 5                    | cache.t4g.medium   |
+| C3        | 6.38           | 12.5                 | cache.m7g.large    |
+| C4        | 12.93          | 12.5                 | cache.m7g.xlarge   |
+| C5        | 26.04          | 15                   | cache.m7g.2xlarge  |
+| C6        | 52.26          | 15                   | cache.m7g.4xlarge  |
+| C7        | 103.68         | 15                   | cache.m7g.8xlarge  |
+| C8        | 209.55         | 30                   | cache.m7g.16xlarge |
+
+
+## Specification
```

Comment:
The specification is almost the same as for the AwsRedisInstance with two additional parameters: **shardCount**, and ****replicasPerShard**, is that correct?